Description: Fix ftbfs with gcc-4.7
 Include <unistd.h> for functions like write(), fork(), etc
Author: Vincent Legout 
Bug-Debian: http://bugs.debian.org/667121
Forwarded: no
Last-Update: 2012-04-04

Index: brutalchess-0.5.2+dfsg/src/faileplayer.cpp
===================================================================
--- brutalchess-0.5.2+dfsg.orig/src/faileplayer.cpp	2006-12-20 21:19:50.000000000 +0100
+++ brutalchess-0.5.2+dfsg/src/faileplayer.cpp		2012-04-04 21:02:32.000000000 +0200
@@ -15,6 +15,8 @@
 #include <sstream>
 #include <string>
 
+#include <unistd.h>
+
 using namespace std;
 
 FailePlayer::FailePlayer()
Index: brutalchess-0.5.2+dfsg/src/xboardplayer.cpp
===================================================================
--- brutalchess-0.5.2+dfsg.orig/src/xboardplayer.cpp	2006-12-19 05:00:01.000000000 +0100
+++ brutalchess-0.5.2+dfsg/src/xboardplayer.cpp	2012-04-04 21:03:30.063044751 +0200
@@ -15,6 +15,8 @@
 #include <sstream>
 #include <string>

+#include <unistd.h>
+
 using namespace std;
 
 